README.md

```diff
@@ -1 +1,34 @@
-# secrethub-examples
\ No newline at end of file
+<p align="center">
+  <img src="https://secrethub.io/img/secrethub-logo.svg" alt="SecretHub" width="380px"/>
+</p>
+<h1 align="center">
+  <i>Examples</i>
+</h1>
+
+[![Version](https://img.shields.io/github/release/secrethub/demo-app.svg)](https://github.com/secrethub/demo-app/releases/latest)
```

Comment:
This points to the wrong repo

Reply:
I don't think we need this badge at all in this repo.